### PR TITLE
Add wheels for boxy and options

### PR DESCRIPTION
**Pull request by HoangGiang93**
I add wheels for donbot

### CODE REVIEW

**Comment by ichumuh** [comment]
did you forget to add a file?
```
$ roslaunch iai_donbot_description upload_with_refills_finger.launch 
... logging to /home/stelter/.ros/log/d87dfed0-0f9c-11eb-8961-04d4c4ace797/roslaunch-ai-00024-19038.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

Undefined parameters [with_wheels,with_boxes]
when instantiating macro: iai_donbot (/home/stelter/iai_ws/src/iai_robots/iai_donbot_description/robots/iai_donbot.xacro)
in file: /home/stelter/iai_ws/src/iai_robots/iai_donbot_description/robots/iai_donbot_with_refills_finger.urdf.xacro
RLException: while processing /home/stelter/iai_ws/src/iai_robots/iai_donbot_description/launch/upload.launch:
Invalid <param> tag: Cannot load command parameter [robot_description]: command [['/opt/ros/melodic/lib/xacro/xacro', '/home/stelter/iai_ws/src/iai_robots/iai_donbot_description/robots/iai_donbot_with_refills_finger.urdf.xacro', 'transmission_hw_interface:=hardware_interface/PositionJointInterface', 'kinematics_config:=/home/stelter/iai_ws/src/iai_robots/iai_donbot_description/config/ur5_calibration.yaml']] returned with code [2]. 

Param xml is <param command="$(find xacro)/xacro '$(arg urdf)' transmission_hw_interface:=$(arg transmission_hw_interface)     kinematics_config:=$(arg kinematics_config)" name="robot_description"/>
The traceback for the exception was written to the log file
```

**Comment by HoangGiang93** [comment]
I fixed the bugs, should I make pull request again or it would be automatically updated?

**Comment by ichumuh** [comment]
it updates automatically, if you push to the same branch, which you did. i'll look at it again soon.

File: iai_donbot_description/robots/iai_donbot_with_refills_finger.urdf.xacro [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   <!-- <link name="map"/> -->
 
-  <xacro:iai_donbot odom="True"/>
+  <xacro:iai_donbot odom="True" with_wheels="True" with_boxes="False" />
```
Comment (ichumuh) [inline iai_donbot_description/robots/iai_donbot_with_refills_finger.urdf.xacro:9]: With boxes should be the default. This is the urdf, i'm using on donbot.

File: iai_boxy_base_2/robots/base.xacro [hunk 2]
```diff
@@ -24,6 +24,106 @@
...
+		
+		<xacro:if value="${with_wheels}">
+			<joint name="wheel_left_up_joint" type="revolute">
+				<limit lower="-1000.0" upper="1000.0" effort="200.0" velocity="1.0" />
+				<origin xyz="0.13 0.09 -0.06" rpy="0.0 0.0 0.0" />
+				<parent link="base_link" />
+				<child link="wheel_left_up" />
+				<axis xyz="0 1 0" />
+			</joint>
+			<link name="wheel_left_up">
```
Comment (ichumuh) [inline iai_boxy_base_2/robots/base.xacro:36]: i think better names would be `wheel_front_left` and so on. up and down don't really make sense.

Comment (ichumuh) [reply iai_boxy_base_2/robots/base.xacro:36]: the wheels also appear red in rviz. Can you color them grey?

Comment (ichumuh) [reply iai_boxy_base_2/robots/base.xacro:36]: the new wheels overlap with the old ones, that are part of the base_link. if the point is, to make donbot pretty, then the base link mesh needs to be cut into pieces

Comment (ichumuh) [reply iai_boxy_base_2/robots/base.xacro:36]: the collision mesh should just be a simple cylinder with the same size as the wheel, to speed up collision checking

**Review by ichumuh — CHANGES_REQUESTED** [review]

**Comment by HoangGiang93** [comment]
I fixed all the issues that @ichumuh mentioned

**Comment by ichumuh** [comment]
look nice, thank!

**Pull request merged**